renovate.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:base"
+    ],
+    "schedule": ["after 10am and before 5pm every weekday"]
```

Comment:
I thought it might be nice to have these come in at a scheduled time when we're not actively developing. 

What do you think?

We can also limit if we only want a certain number at a time

Reply:
oh that's a nice feature. Actually, I would prefer the opposite so I don't see them when I'm working XD 
either way is fine by me.

for cadence, is there a way to limit what it updates? only major versions or security issues?

Reply:
I figured we won't be working on this project when we are at our day jobs. So I set that schedule.

Reply:
There is an extended config doc here: https://docs.renovatebot.com/configuration-options/#schedule

Feel free to open a PR if you'd like to adjust it

Reply:
Oh you mean you don't want an alert during your day job... I misread. Let's do once a week, like Monday night or something